frontend/src/components/CardTitle/CardTitle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Typography,} from "@material-ui/core";
+import {APP_NAME} from "../../constants/constants";
+import {VerticalSpace} from "../Shared/VerticalSpace"
+import {Centered} from "../Shared/Centered";
+import styled from "styled-components";
+
+export type TitleProps = {
+    readonly title: string | undefined;
+};
+
+const CardTitleComponent = (props: TitleProps | undefined) => {
+    const titleComponent = (
+        <>
```

Comment:
To się przydaje tylko wtedy, jeśli chcesz zwrócić jako komponent więcej niż jeden tag. Tutaj mamy tag główny - Centered, więc to nic nie wnosi.